### ISSUE TITLE
TypeError: Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by yoitsro** [evanw/node-source-map-support#260]
I'm getting this error when executing my code:

```
/home/app/node_modules/source-map-support/source-map-support.js:379
return state.nextPosition.name || originalFunctionName();
^
TypeError: Cannot read property 'name' of undefined
```

It looks like the [previous line](https://github.com/evanw/node-source-map-support/blob/v0.5.15/source-map-support.js#L376) checks to see if `state.nextPosition` is `null`, but does not check if it's `undefined`.

**Comment by 6utt3rfly** [comment]
I can confirm that I found this issue too I'm rolling back from 0.5.15 to 0.5.13 for now

**Comment by yoitsro** [comment]
Annoyingly, it's one of my downstream packages which is causing this issue for me. In particular, `@google-cloud/trace-agent`: https://github.com/googleapis/cloud-trace-nodejs/blob/master/src/util.ts#L265

**Comment by rvillane** [comment]
@kjin any chances @google-cloud/trace-agent can pin source-map-support version ?

**Comment by kjin** [comment]
You can pin it yourself by adding your own `source-map-support` dependency on `0.5.13` exactly, npm should de-dupe it: https://gist.github.com/kjin/1b0ec56149f9db708922460f66a94df0

**Comment by yoitsro** [comment]
@kjin good point! Thanks! For the sake of others running into this problem, would it make sense to pin it directly in your package directly anyway?

**Comment by LinusU** [comment]
Was out traveling today so got to this late, fixing this now 🐎

**Comment by LinusU** [comment]
Published as `0.5.16` 🚀
